lib/fastlane/actions/add_icon_overlay.rb

```diff
@@ -0,0 +1,181 @@
+                                       env_name: "FL_ADD_ICON_OVERLAY_TEXT_FOREGROUND_COLOR",
+                                       description: "Foreground color for the text overlay (Text color). If a hex value is provided must contain the '#' character",
+                                       is_string: true,
+                                       default_value: "white",
+                                       optional: true
+                                      ),
+          FastlaneCore::ConfigItem.new(key: :text_overlay_height,
+                                       env_name: "FL_ADD_ICON_OVERLAY_OVERLAY_HEIGHT",
+                                       description: "Height of the overlay in respect to the overall height of the icon (e.g. 0.5 for 50%)",
+                                       optional: true,
+                                       default_value: 0.3,
+                                       is_string: false,
+                                       verify_block: Proc.new do |value|
+                                         raise "Invalid height percentage provided. Must be a float value between 0.0 & 1.0" unless value and value.to_f > 0 and value.to_f <= 1.0
+                                       end),
+          FastlaneCore::ConfigItem.new(key: :text_overlay,
+                                       description: "The text to be overlayed over the icon. Can contain '\\t' and '\\n' when 'overlay_type' is 'text', not on 'banner'",
+                                       is_string: true,
+                                       optional: true
+                                      ),
+          FastlaneCore::ConfigItem.new(key: :text_overlay_position,
+                                       env_name: "FL_ADD_ICON_OVERLAY_TEXT_OVERLAY_POSITION",
+                                       description: "Where on the icon the text will be overlayed (south & north). Only available for 'overlay_type' 'text'",
+                                       is_string: true,
+                                       optional: true,
+                                       default_value: "south",
+                                       verify_block: Proc.new do |value|
+                                         raise "Invalid text overlay position. Expected 'north' or 'south' and '#{value}' provided" unless value and (value == "north" || value == "south")
+                                       end),
+          FastlaneCore::ConfigItem.new(key: :text_overlay_font,
+                                       env_name: "FL_ADD_OVERLAY_ICON_FONT",
+                                       description: "Name of the font to be used on the text overlay. Must be in font list `convert -list font or full path to font",
+                                       is_string: true,
+                                       optional: true,
+                                       default_value: "Helvetica"
+                                       )
```

Comment:
Already fixed with e0e2bb3f35cfd00ad9401eea4a7e4b72dffcb87a but `HoundCI` failed to pick the change